src/boards/include/boards/pimoroni_servo2040.h

```diff
@@ -0,0 +1,236 @@
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+#ifndef _BOARDS_PIMORONI_SERVO2040_H
+#define _BOARDS_PIMORONI_SERVO2040_H
+
+// For board detection
+#define PIMORONI_SERVO2040
+
+// --- BOARD SPECIFIC ---
+#ifndef SERVO2040_SERVO_1_PIN
+#define SERVO2040_SERVO_1_PIN 0
+#endif
+
+#ifndef SERVO2040_SERVO_2_PIN
+#define SERVO2040_SERVO_2_PIN 1
+#endif
+
+#ifndef SERVO2040_SERVO_3_PIN
+#define SERVO2040_SERVO_3_PIN 2
+#endif
+
+#ifndef SERVO2040_SERVO_4_PIN
+#define SERVO2040_SERVO_4_PIN 3
+#endif
+
+#ifndef SERVO2040_SERVO_5_PIN
+#define SERVO2040_SERVO_5_PIN 4
+#endif
+
+#ifndef SERVO2040_SERVO_6_PIN
+#define SERVO2040_SERVO_6_PIN 5
+#endif
+
+#ifndef SERVO2040_SERVO_7_PIN
+#define SERVO2040_SERVO_7_PIN 6
+#endif
+
+#ifndef SERVO2040_SERVO_8_PIN
+#define SERVO2040_SERVO_8_PIN 7
+#endif
+
+#ifndef SERVO2040_SERVO_9_PIN
+#define SERVO2040_SERVO_9_PIN 8
+#endif
+
+#ifndef SERVO2040_SERVO_10_PIN
+#define SERVO2040_SERVO_10_PIN 9
+#endif
+
+#ifndef SERVO2040_SERVO_11_PIN
+#define SERVO2040_SERVO_11_PIN 10
+#endif
+
+#ifndef SERVO2040_SERVO_12_PIN
+#define SERVO2040_SERVO_12_PIN 11
+#endif
+
+#ifndef SERVO2040_SERVO_13_PIN
+#define SERVO2040_SERVO_13_PIN 12
+#endif
+
+#ifndef SERVO2040_SERVO_14_PIN
+#define SERVO2040_SERVO_14_PIN 13
+#endif
+
+#ifndef SERVO2040_SERVO_15_PIN
+#define SERVO2040_SERVO_15_PIN 14
+#endif
+
+#ifndef SERVO2040_SERVO_16_PIN
+#define SERVO2040_SERVO_16_PIN 15
+#endif
+
+#ifndef SERVO2040_SERVO_17_PIN
+#define SERVO2040_SERVO_17_PIN 16
+#endif
+
+#ifndef SERVO2040_SERVO_18_PIN
+#define SERVO2040_SERVO_18_PIN 17
+#endif
+
+#ifndef SERVO2040_NUM_SERVOS
+#define SERVO2040_NUM_SERVOS 18
+#endif
+
+#ifndef SERVO2040_LED_DAT_PIN
+#define SERVO2040_LED_DAT_PIN 18
+#endif
+
+#ifndef SERVO2040_NUM_LEDS
+#define SERVO2040_NUM_LEDS 6
+#endif
+
+#ifndef SERVO2040_I2C
+#define SERVO2040_I2C 0
+#endif
+
+#ifndef SERVO2040_INT_PIN
+#define SERVO2040_INT_PIN 19
+#endif
+
+#ifndef SERVO2040_SDA_PIN
+#define SERVO2040_SDA_PIN 20
+#endif
+
+#ifndef SERVO2040_SCL_PIN
+#define SERVO2040_SCL_PIN 21
+#endif
+
+#ifndef SERVO2040_USER_SW_PIN
+#define SERVO2040_USER_SW_PIN 23
+#endif
+
+#ifndef SERVO2040_ADC_ADDR_0_PIN
+#define SERVO2040_ADC_ADDR_0_PIN 22
+#endif
+
+#ifndef SERVO2040_ADC_ADDR_1_PIN
+#define SERVO2040_ADC_ADDR_1_PIN 24
+#endif
+
+#ifndef SERVO2040_ADC_ADDR_2_PIN
+#define SERVO2040_ADC_ADDR_2_PIN 25
+#endif
+
+#ifndef SERVO2040_A0_PIN
+#define SERVO2040_A0_PIN 26
+#endif
+
+#ifndef SERVO2040_A1_PIN
+#define SERVO2040_A1_PIN 27
+#endif
+
+#ifndef SERVO2040_A2_PIN
+#define SERVO2040_A2_PIN 28
+#endif
+
+#ifndef SERVO2040_NUM_ADC_PINS
+#define SERVO2040_NUM_ADC_PINS 3
+#endif
+
+#ifndef SERVO2040_SHARED_ADC_PIN
+#define SERVO2040_SHARED_ADC_PIN 29
+#endif
+
+#ifndef SERVO2040_SENSOR_1_ADDR
+#define SERVO2040_SENSOR_1_ADDR 0b000
```

Comment:
Not that it matters, but any particular reason why these are specified in binary?

Reply:
It was just convenience for my mental processing of the 3 bits (and thus pins) that are being set.

Reply:
As I said, it doesn't matter, so feel free to leave them as-is.